docker-compose.yml

```diff
@@ -127,6 +128,28 @@ services:
 
   solana-test-validator:
     image: audius/solana-programs:latest
+    # NOTE: Building solana program can take a lot of time especially on m1 (upto 30 mins) so we used pre-built images
```

Comment:
Do we need this?

https://github.com/AudiusProject/audius-protocol/tree/main/solana-programs#building-on-apple-m1

I think this instructions accomplish what we need 

also build on m1 takes like 12+ hours sometimes


Let's standardize around either using the compose file to build or using a custom README+script

Reply:
Those instructions do not work anymore.

On my M1, with a complete clean slate (after `docker system prune -a` and new protocol clone) installing together with anchor takes ~30 mins, without anchor ~20 mins, if it takes longer than that it is likely due to old version of docker.

I think it is possible that local builds are affecting this, and you should try a new clone if that is the case.

After Isaac's change the Dockerfile contains syntax errors and does not build, the 30 mins I got was with the bugs fixed, I pushed the dev Dockerfile that I used.